File: 0796-rotate-string/0796-rotate-string.js
/**
 * @param {string} s
 * @param {string} goal
 * @return {boolean}
 */
var rotateString = function(s, goal) {
    s= s+s;
   
    if (s.includes(goal)){
        return true
    }

    return false

   
    
};

File: 0796-rotate-string/0796-rotate-string.cpp
class Solution {
public:
    bool rotateString(string s, string goal) {
        if(s==goal) return true;
        if(s.size()!=goal.size()) return false;
        string temp = s+s;
        int n = goal.size();
        for(int i=0;i<n;i++){
            if(temp.substr(i,n)==goal){
                return true;
            }
        }
        return false;
    }
};


